x-pack/plugins/infra/server/plugin.ts

```diff
@@ -82,9 +84,17 @@ export interface InfraPluginSetup {
 export class InfraServerPlugin implements Plugin<InfraPluginSetup> {
   public config: InfraConfig;
   public libs: InfraBackendLibs | undefined;
+  public logger: Logger;
+
+  private logsRules: RulesService;
+  private metricsRules: RulesService;
 
   constructor(context: PluginInitializerContext) {
     this.config = context.config.get<InfraConfig>();
+    this.logger = context.logger.get();
+
+    this.logsRules = new RulesService('observability.logs', this.logger.get('logsRules'));
```

Comment:
This hardcoded string is used in one more place. Let's move it to the constants file I created recently in the common folder

Reply:
The only other location duplicating this is one of the hard-coded RBAC datastructures, which will hopefully not stay that way. I hope we can de-duplicate this when the registration API changes.